src/objects/Account.object

```diff
@@ -1,13 +1,253 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
     <fields>
-        <fullName>MyExtID__c</fullName>
```

Comment:
Why remove this "MyExtID__c" field from the Account object. it is used in the previous assignment. it should be unchanged/untouched for this assignment.